docs/multiform.rst

```diff
@@ -60,6 +60,13 @@ use an OrderedDict to define the :attr:`~MultiForm.form_classes`. ::
             ('profile', ProfileForm),
         ))
 
+An implementation of the attribute :attr:`~MultiForm.base_fields` is provided,
+but the keys are a combination of the subform keys and the field names rather
+than just the field names. The reason an implemention is provided at all, given
+it is merely an approximation, is interoperability with ``WizardView``, which
+introspects the values of :attr:`~MultiForm.base_fields`.  See
+:ref:`wizardview_guide`.
+
```

Comment:
My thinking was its essentially nothing to do with MultiForm, it's really between Django Forms and WizardView -- but I guess it could help someone who hasn't read the WizardView documentation. I've added a similar message, but made it a little more explicit about when exactly the problems might happen.